specification/ingest/get_ip_location_database/GetIpLocationDatabaseResponse.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { DatabaseConfigurationFull } from '@ingest/_types/Database'
+import { Id } from '@_types/common'
+import { long } from '@_types/Numeric'
+import { EpochTime, UnitMillis } from '@_types/Time'
+
+export class Response {
+  body: { databases: DatabaseConfigurationMetadata[] }
+}
+
+class DatabaseConfigurationMetadata {
+  id: Id
+  version: long
+  modified_date_millis: EpochTime<UnitMillis>
```

Comment:
the timestamp can be one of `modified_date` or `modified_date_millis` depending on the `humanReadable`flag. [server code](https://github.com/elastic/elasticsearch/blob/bb8503a2521a31b73360a86c62ccec3047369b26/modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/direct/GetDatabaseConfigurationAction.java#L113). if I'm reading the `UNIX_EPOCH_MILLIS_FORMATTER` correctly, the types should be:
```
modified_date_millis?: string
modified_date?: EpochTime<UnitMillis>
```

Reply:
```suggestion
  modified_date_millis?: EpochTime<UnitMillis>
  modified_date?: EpochTime<UnitMillis>
```